quake-terminal@diegodario88.github.io/quake-mode.js

```diff
@@ -416,14 +439,23 @@ export const QuakeMode = class {
           (area.width - terminalWidth) / horizontalAlignmentSettingsValue
       );
 
-    this.terminalWindow.move_to_monitor(monitorDisplayScreenIndex);
+    return new Mtk.Rectangle(terminalX, area.y, terminalWidth, terminalHeight);
```

Comment:
Actually, should have been:
```suggestion
    return Mtk.Rectangle.new(terminalX, area.y, terminalWidth, terminalHeight);
```